server/api/users.js

```diff
@@ -53,3 +53,68 @@ router.get('/:id/latestguesses', async (req, res, next) => {
 		next(error)
 	}
 })
+
+const getRandomInt = max => {
+	return Math.floor(Math.random() * max)
+}
+
+const completedPrevGameOrStartNewGame = async (req, res, next) => {
+	try {
+		let latestWordle = await WordleGame.findAll({
+			where: { userId: req.params.id },
+			include: AcceptedGuess
+		})
+		latestWordle ? (latestWordle = latestWordle[latestWordle.length - 1]) : null
+		if (!latestWordle || latestWordle.acceptedGuesses.length === 5) {
+			next()
+		} else {
+			console.log('not finished game yet')
+		}
+	} catch (error) {
+		next(error)
+	}
+}
+
+//create a new target word and asign that to a new game - send back the new target word to client
+router.get(
+	'/:id/newTargetWord',
+
+	completedPrevGameOrStartNewGame,
+	async (req, res, next) => {
+		try {
+			const randomWord = await AcceptedWord.findByPk(getRandomInt(10656))
+			const targetWord = await TargetWord.create({ content: randomWord.content })
```

Comment:
See previous note, but I think ultimately that TargetWord should be it's own table with a one to Many relationship with WordleGame (every wordle game has one target word, but target words can be in multiple wordle games).